langchain4j-mistral-ai/src/main/java/dev/langchain4j/model/mistralai/DefaultMistralAiHelper.java

```diff
@@ -0,0 +1,121 @@
+package dev.langchain4j.model.mistralai;
+
+import dev.langchain4j.data.message.ChatMessage;
+import dev.langchain4j.data.message.ChatMessageType;
+import dev.langchain4j.model.output.FinishReason;
+import dev.langchain4j.model.output.TokenUsage;
+import okhttp3.Response;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+import static dev.langchain4j.internal.Utils.isNullOrBlank;
+import static dev.langchain4j.model.output.FinishReason.*;
+import static java.util.stream.Collectors.toList;
+
+public class DefaultMistralAiHelper{
```

Comment:
```suggestion
class DefaultMistralAiHelper {
```
Please make sure all internal implementation classes that user should not see are package-private